electron/main/index.ts

```diff
@@ -95,6 +95,20 @@ Erc20Token.createTokenConstructors().forEach(({ name, coinConstructor }) => {
   sdk.register(name, coinConstructor);
 });
 
+function handleSdkError(e: unknown): string {
+  if (typeof e === 'string' && e !== null) {
```

Comment:
@zahin-mohammad you're already checking that its a string, why check if its not null?

Reply:
fair, the null check isn't needed here.